build.proj

```diff
@@ -46,32 +47,36 @@
     <Message Text="Dont remove this target" />
   </Target>
 
-  <Target Name="BuildDotnetCliBuildFramework" 
-          Inputs="@(DotnetCliBuildFrameworkInputs)" 
+  <Target Name="BuildDotnetCliBuildFramework"
+          Inputs="@(DotnetCliBuildFrameworkInputs)"
           Outputs="$(CLIBuildDll)"
           DependsOnTargets="MSBuildWorkaroundTarget">
     <PropertyGroup>
       <DotnetCliBuildDirectory>$(MSBuildThisFileDirectory)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
     </PropertyGroup>
 
+    <ItemGroup>
```

Comment:
Can this get moved to its own target? seems like a wholely independent intention...